linera-indexer/tests/test.rs

```diff
@@ -96,38 +48,26 @@ fn indexer_running(child: &mut Child) {
     }
 }
 
-async fn request<T, V>(client: &reqwest::Client, url: &str, variables: V) -> T::ResponseData
-where
-    T: GraphQLQuery<Variables = V> + Send + Unpin + 'static,
-    V: Send + Unpin,
-{
-    let response = post_graphql::<T, _>(client, url, variables).await.unwrap();
-    response
-        .data
-        .unwrap_or_else(|| panic!("{:?}", response.errors))
-}
-
 async fn transfer(client: &reqwest::Client, from: ChainId, to: ChainId, amount: f32) {
```

Comment:
I'd make the `amount` a `&str`. Using floating-point numbers can lead to rounding-errors, which we should avoid.